src/component/DefaultTooltipContent.tsx

```diff
@@ -127,7 +127,7 @@ export class DefaultTooltipContent<TValue extends ValueType, TName extends NameT
     const wrapperCN = classNames('recharts-default-tooltip', wrapperClassName);
     const labelCN = classNames('recharts-tooltip-label', labelClassName);
 
-    if (hasLabel && labelFormatter) {
+    if (hasLabel && labelFormatter && !_.isNil(payload)) {
```

Comment:
I think it would be better to just compare like

```javascript
payload === null
```

Instead of using loadsh

Reply:
Fair enough. Lodash was already being used, but I have no problem with removing this dependency on another package.

Reply:
Fair enough. The package was already using lodash, but I have no problem reducing reliance on other package for this.

Reply:
For what it is worth, this line of code was just 5 lines above it:
`const hasLabel = !_.isNil(label);`
I'm not going to reach out of scope and removing lodash elsewhere, but just so that we are on the same page about the usage of lodash in this package.